internal/services/datadog/datadog_monitor_tag_rule_resource.go

```diff
@@ -363,3 +370,27 @@ func flattenFilteringTags(input *[]rules.FilteringTag) []interface{} {
 	}
 	return results
 }
+
+func isDefaultSettings(input *rules.MonitoringTagRules) bool {
+	if input == nil {
+		return false
+	}
+
+	if input.Name == nil || !strings.EqualFold(*input.Name, "default") {
+		return false
+	}
```

Comment:
I think this should be removed? The `name` property defaults to `default`, so could have other settings that have changed thus making this an incorrect assumption?

Reply:
Yes. It is removed.